beacon-chain/verification/data_column.go

```diff
@@ -473,7 +473,7 @@ func (dv *RODataColumnsVerifier) SidecarKzgProofVerified() (err error) {
 		return columnErrBuilder(errors.Wrap(err, "verify data column commitment"))
 	}
 
-	dataColumnBatchKZGVerificationHistogram.Observe(float64(time.Since(startTime).Milliseconds()))
+	DataColumnBatchKZGVerificationHistogram.Observe(float64(time.Since(startTime).Milliseconds()))
```

Comment:
Can we annotate the histogram with labels (`batch`, `fallback`)? The two code paths being measured have meaningful differences in a debugging context so it seems helpful to be able to split them out.